aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.kt

```diff
@@ -99,6 +99,11 @@ class AWSCognitoAuthPlugin : AuthPlugin<AWSCognitoAuthService>() {
         return pluginConfigurationJSON
     }
 
+    @InternalAmplifyApi
+    fun addToUserAgent(key: String, value: String) {
```

Comment:
Is `addToUserAgent` meant to be called before each API call or once during configure?

Reply:
Once during configure, this will add it to all future Auth calls